guides/authn_authz/scopes.md

```diff
@@ -419,6 +423,8 @@ config :my_app, :scopes,
     module: MyApp.Accounts.Scope,
     assign_key: :current_scope,
     access_path: [:organization, :id],
+    route_prefix: "/orgs/:org",
+    route_access_path: [:organization, :slug],
```

Comment:
In this example, do we want to implement Phoenix.Param or use `route_access_path`? I would prefer Phoenix.Param, it feels more idiomatic.

Reply:
We can remove this if we use Enum.drop, so that gets my vote :)